### DISCUSSION TITLE
ESP32 - externe WLAN-Antenne aktivieren / Antennenprobleme

### DISCUSSION THREAD

**Discussion by Zurrmaxe** [hoylabs/OpenDTU-OnBattery#1469] [General]
Hallöchen zusammen

Für einige ist das nachstehende ein alter Hut, für andere vieleicht jedoch die Lösung eines WLAN-Problemes:

Bei einigen ESP32-Modulen werden externe Antennenanschlüsse angeboten.  
Sofern noch zusätzlich eine WLAN-Antenne auf dem Board vorhanden ist, muss der externe Antennenanschlüsse häufig erst aktiviert bzw. umgeschaltet werden. 
Beide gleichzeitig funktioniert meist nicht !
Gerade die neuen S3-Module haben oft einen solchen Anschluss.
Ausgeliefert werden sie in der Regel mit der aktiverten internen WLAN-Antenne ! 
Steckt man eine externe Antenne in die Buchse, funktioniert sie nicht. 
Diese Antenne ist quasi nur ein "Dummy".  Daher ist der Antennenanschluss umzuschalten.

Hier einmal ein (negatives) Beispiel eines Herstellers:
![WLAN](https://github.com/user-attachments/assets/385de9fb-a432-4d9f-a7a2-b35d57add07c)

Bei diesem neuen Modul war gar keine Antenne aktiviert.  Reichweite nur ein paar Meter bis zur Fritzbox.
Man könnte dann schnell vermuten, dass das WLAN defekt ist. 
Ist es aber nicht. Nur keine Antenne angeschlossen !

![ipex](https://github.com/user-attachments/assets/d99aac57-924c-421c-b612-cca3ad79ef85)

So wird es richtig gemacht. Jetzt ist der Anschluss aktiviert.

Also Augen auf beim Chipeinkauf.

Hierzu verweise ich auch auf folgende weiterführenden Informationen zum Lesen:

https://randomnerdtutorials.com/esp32-cam-connect-external-antenna/
und
https://nachbelichtet.com/esp32-cam-probleme-beheben

Viel Spass beim Lesen des Artikels.
Alf

**Comment by stefan123t** [comment]
Hallo @Zurrmaxe ja man muss die Lötbrücke bzw den 0 Ohm Widerstand richtig einlöten. Meist brückt er R15 und somit funktioniert die on Board PCB Antenne. Wenn man den IPEX Anschluss nutzen will muss man diese(n) Brücke / „Widerstand“ auslöten und in Position R14 wieder einlöten. Dann erst ist der externe Antennenanschluss aktiviert und man muss nur noch eine passende Stummelantenne daran anschließen.

**Reply by Zurrmaxe** [reply]
Hallo stefan123t
Auch ich bin geschickt im Löten, aber diese kleinen Lötstellen sind echt eine Herausforderung und können zur Zerstörung des ESP32 führen. Daher besser gleich das richtiges Modul einkaufen. 
Es werden vermutlich nicht alle Nutzer der OpenDTUs in der Lage sein, diese kleinen Teilchen umzulöten. 
Dazu braucht man eine spezielle Spitze bzw. Lötnadel und ganz wenig Watt....

Ich habe mal alle meine ESP32 mit IPEX Anschluss durchgescheckt. 

Sie sind nicht aktviert, wenn eine interne WLAN-Antenne vorhanden ist.

Somit kaufe ich zukünftig nur noch Module ohne eine interne WLAN-Antenne !

gruss Alf.

**Reply by Rustimation** [reply]
Hast du einen Lieferanten gefunden, der die ESP32-Boards umgebaut (ext. antenna ready) verkauft? Ich nicht.
Danke
Chris

**Reply by knopers1** [reply]
<img width="1346" height="457" alt="11" src="https://github.com/user-attachments/assets/4e8393cc-c39e-4799-890b-d308cce3fabe" />

**Reply by Rustimation** [reply]
Danke. Ich hatte vergessen zu erwähnen, dass ich ein ESP32**-Cam** Modul suche, das entsprechend für eine externe Antenne modifiziert ist. Ich hab's nach drei zerstörten Boards immer noch nicht hinbekommen :-(
Mediarath.de bietet sowas an (Board, Programmierdock, Antenne, umgelötet), aber für ca 25€ sehr teuer.
Viel Spaß noch
Chris

**Reply by Marc9485** [reply]
@Rustimation 3 esp's zu schrotten ist auch nicht viel günstiger gewesen 😉

**Reply by Sorbat** [reply]
> Danke. Ich hatte vergessen zu erwähnen, dass ich ein ESP32**-Cam** Modul suche, das entsprechend für eine externe Antenne modifiziert ist. Ich hab's nach drei zerstörten Boards immer noch nicht hinbekommen :-(
> 
> Mediarath.de bietet sowas an (Board, Programmierdock, Antenne, umgelötet), aber für ca 25€ sehr teuer.
> 
> Viel Spaß noch
> 
> Chris

Warum tauschst du den esp nicht aus?
